src/plugins/explore/public/application/utils/state_management/utils/redux_persistence.ts

```diff
@@ -266,6 +272,7 @@ const getPreloadedQueryEditorState = async (
     },
     promptModeIsAvailable,
     promptToQueryIsLoading: false,
+    summaryAgentIsAvailable: false,
     editorMode,
     lastExecutedPrompt: '',
   };
```

Comment:
there is an associated `redux_persistence.test.ts` file, lets update that with this change